lms/templates/main_django.html

```diff
@@ -28,14 +28,14 @@
   <div class="window-wrap" dir="${static.dir_rtl()}">
     <a class="nav-skip" href="{% block nav_skip %}#content{% endblock %}">{% trans "Skip to main content" %}</a>
     {% with course=request.course %}
-      {% include "header.html" %}
+      {% include "header.html"|microsite_template_path %}
```

Comment:
@mattdrayer @chrisndodge I investigated it on my side after pulling down this branch. It seems me, wiki page is based on Django templates rather than mako and after running the lms on this branch , It seems me header from microsite is not loading in wiki. You can see that edx-microsite/mit-prof-ed/templates/header.html is based on mako
wiki page is trying to load this microsite header.html